src/coreclr/pal/src/misc/jitsupport.cpp

```diff
@@ -82,7 +82,7 @@ static const CpuCapability CpuCapabilities[] = {
 // If the capability name is not recognized or unused at present, zero is returned.
 static unsigned long LookupCpuCapabilityFlag(const char* start, size_t length)
 {
-    for (int i = 0; i < _countof(CpuCapabilities); i++)
+    for (int i = 0; i < (int)_countof(CpuCapabilities); i++)
```

Comment:
I would avoid casting
```suggestion
    for (size_t i = 0; i < _countof(CpuCapabilities); i++)
```